.github/workflows/ci.yml

```diff
@@ -19,32 +17,40 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ "ubuntu-latest", "macos-latest"]
-        python-version: [ "3.7", "3.8", "3.9", "3.10" ]
+        os: [ "ubuntu-latest", "macos-latest", "macos-14" ]
```

Comment:
Yeah, I found it a bit surprising as well that there wasn't Windows testing active in here.  Especially since there are scripts for the build and it's on conda-forge.  

I did start to add that in the first time around, but then opted to limit the scope of changes (w/ my limited understanding of the Windows build).  I can give it a try though - I agree it would be good to have.

Reply:
Yeah, maybe just add it to the matrix and add the corresponding "conda setup" step for it, and let's see how it goes. If it doesn't succeed right away, we can just revert it and plan to deal with it alone after this PR is merged.

Reply:
Well, I think the Windows builds are for another day.  

I've tried:
* Different build scripts (MSVC/OMP - tests fail, OMP - build fails, MSVC/no OMP -  tests fail IIRC)
* The old Windows environment (tests fail) and the new universal env file (tests fail + some errors in the build that I didn't notice in the other runs that were ignored)

It looks like the feedstock repo effectively uses the OMP (non-MSVC) build script, but the environment is a bit different.  Looking into that is probably the next step.

For now, I'm just going to pull Windows builds out of the CI runs and we can add those in later.

Reply:
Yeah, that sounds good; thanks a lot for looking into it!